## AMRnet

This is the home for all code relating to the AMRnet project


### What is AMRnet?
AMRnet will be a data-visualisation dashboard that makes genome-derived antimicrobial resistance (AMR) data accessible to a wide range of stakeholders including policy makers. 

The concept is based on the [TyphiNET](https://typhi.net) dashboard, which serves data on the typhoid fever pathogen _Salmonella_ Typhi.

### Why is AMR surveillance important?
AMR surveillance involves gathering data on the prevalence of resistance to specific antimicrobials amongst infections caused by specific pathogens. AMR surveillance is recognised by the World Health Organisation as an essential tool to inform policies and infection prevention and control responses.

Policy makers use the data to inform on a range of clinical and public health issues including recommendations for the use of drugs for empirical treatment or prevention of infection, and policies affecting availability of requisite drugs; and making the case for national immunisation programs (as lack of reliable treatment for incident infections can tip the balance in favour of population-scale preventative measures).

### Why is genomics important?
Pathogen sequencing is increasingly used as a primary or core method for pathogen characterisation in both research and public health microbiology laboratories. Laboratory protocols for sequencing are relatively easy to standardise and the resulting sequence data, being digital, is straightforward to store and share. 

Genome data can simultaneously provide information on all known AMR mechanisms for all drugs of interest, together with variant/subtype information such as lineage, serotype or pathotype which can have additional epidemiological or clinical relevance. Analyses at gene/mechanism level are also more sensitive to change than analysis of resistance phenotypes, thus have the potential to detect and respond to changing trends much earlier.

### What will the AMRnet project do?
#### There is untapped potential for pathogen genomic data to fill gaps in AMR surveillance, that could inform policies and responses to the global AMR crisis and help reduce future disease burden.

Development of genomic databases and digital infrastructure has so far, rightly, been geared towards supporting basic microbiology and public health investigations (e.g. detecting outbreaks and tracking the emergence and spread of AMR mechanisms), and there are no tools specifically designed to mine these data for the purpose of extracting and making accessible AMR surveillance-ready information to a high-level audience of policy makers. This is particularly relevant to low- and middle-income countries that have a high AMR burden but lack routine diagnostics or national surveillance programs, yet there are untapped genomic data from local research studies or from travellers returning to countries with routine genomic surveillance, which could be used to estimate national annual prevalence to help understand local drivers and mechanisms of resistance and inform policy. While there is already an established culture of pathogen sequence data sharing, these data are essentially invisible to policy makers, and are hard to distinguish amongst the broader set of genome data that is not relevant to surveillance.

We aim to address this by making high-quality, robust and reliable genome-derived AMR surveillance data accessible to a wide audience, via an open-source AMRnet dashboard. AMRnet visualisations will be geared towards showing national annual prevalence estimates and trends, that can be broken down and explored in terms of underlying genotypes and AMR mechanisms, for bacterial pathogens of public health importance.

This will be achieved by developing integrated solutions to multiple related challenges, including:
* improving the availability and identifiability of relevant data (including sequence, source and purpose-of-sampling descriptors to ensure only surveillance-ready data are included in prevalence estimates); 
* filling in gaps in genome data analysis & interpretation (e.g. organism-specific interpretive frameworks for AMR genotyping); and 
* making data accessible through interactive visualisation tools and APIs
